CONTRIBUTING.md

```diff
@@ -97,10 +101,10 @@ submitting a fix (even better :star2:).
 You can submit fixes to the documentation pages completely online without having to
 download and install anything:
 
-* On each documentation page, there should be an "Improve This Page" link at the very
+* On each documentation page, there should be an "Suggest edit" link at the very
```

Comment:
* On each documentation page, there should be ~~an~~ a "Suggest edit" link at the very